net/relayd/src/opnsense/service/templates/OPNsense/Relayd/relayd.conf

```diff
@@ -56,7 +56,7 @@ table <{{ table.name }}>{{ disable }} {
 
 {% if helpers.exists('OPNsense.relayd.virtualserver') %}
 {%    for virtualserver in helpers.toList('OPNsense.relayd.virtualserver') %}
-{%      if virtualserver.transport_table in all_active_tables %}
+{%      if virtualserver.transport_table in all_active_tables or virtualserver.backuptransport_table in all_active_tables in all_active_tables %}
```

Comment:
`in all_active_tables` at the end is to much

Reply:
oops, will fix in next commit